src/modules/layouts/SiteContext.js

```diff
@@ -71,6 +72,38 @@ const SiteProvider = ({ children, value }) => {
       }
     })
 
+
+    //Map for "Has" filters.
+    //Checks if exists filters with their regex match 
+    //has a quantifiable amount to suggest an entry "Having [X]" 
+    ExistsFilters.map(({fragment, regex}) => {
+      if (node.rawBody.includes(`<${fragment}>`)) {
+        const matches = node.rawBody.match(regex); //Regex match based on fragment
```

Comment:
If we're not going to use the capture groups and are just looking for a boolean, you could use `regex.test(node.rawBody)` I think and skip the length check below!